.travis.yml

```diff
@@ -13,12 +13,11 @@ env:
   global:
   - RAILS_ENV=test
 
-before_install:
-- docker pull solr:7.4.0
-- docker run -d -p 8983:8983 -v "$(pwd)"/solr/conf:/myconfig solr:7.4.0 solr-create -c blacklight-core -d /myconfig
-- docker ps -a
-
 before_script:
+- bundle exec rails docker:pull
+- bundle exec rails docker:run_ci
+- bundle exec rails docker:build
+- docker ps -a
```

Comment:
@cdmo do we need this? it just displays the running container, right?

Reply:
it has been there all along... not sure why it's showing it as added. But, anyway, it's not really needed no :)